src/components/Map/Map.tsx

```diff
@@ -456,6 +463,21 @@ export default function Map(props: MapProps): JSX.Element {
           {mapSources}
           <NavigationControl showCompass={false} style={navControlStyle} position='bottom-right' />
           <AttributionControl compact={true} style={{ marginRight: '5px' }} />
+          <Box
+            style={{
+              width: 27,
+              backgroundColor: `${theme.palette.TwClrBaseWhite}`,
+              position: 'absolute',
+              bottom: '30%',
```

Comment:
Curious.. do we know how the mobile app does this?

you might be able to use absolute pixels here, maybe? see what the height is up to the info button perhaps?

30% is not going to scale well in large screens (see image)

<img width="805" alt="Terraware App 2023-07-27 16-49-49" src="https://github.com/terraware/terraware-web/assets/1865174/25c3a564-3c2a-4a14-ad72-cd1f86f1aa55">

Reply:
True, I just checked in mobile and it is done same way (with absolute positioning). Will replace it with pixels